clients/src/main/java/org/apache/kafka/clients/consumer/internals/OffsetsRequestManager.java

```diff
@@ -268,7 +326,7 @@ private CompletableFuture<ListOffsetResult> buildListOffsetRequestToNode(
                 .forConsumer(requireTimestamps, isolationLevel, false)
                 .setTargetTimes(ListOffsetsRequest.toListOffsetsTopics(targetTimes));
 
-        log.debug("Creating ListOffsetRequest {} for broker {} to reset positions", builder,
```

Comment:
I really think ListOffsetRequest is fine.

Reply:
I'm trying to use the proper API key for that request, as defined in the Kafka protocol